cosmic_ray/worker.py

```diff
@@ -58,21 +62,44 @@ def worker(module_name,
     """
     try:
         module = importlib.import_module(module_name)
+        module_source_file = inspect.getsourcefile(module)
```

Comment:
This is brilliant! I did not realize that `inspect` had functions like `getsourcefile()` and `getsource()`. I think I can use these in other parts of CR.